_posts/2024-03-07-cache-refresh.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title:  "A cache refresh for Trino"
+author: Manfred Moser
+image: /assets/blog/trino-cache-refresh.png
+excerpt_separator: <!--more-->
+show_pagenav: true
+---
+
+Thinking about our recent work on caching in Trino reminds me of the famous
+saying, ["There are only two hard things in computer science: cache invalidation
+and naming things](https://www.karlton.org/2017/12/naming-things-hard/)." Well,
+in the Trino community we know all about caching and naming. With the recent
+[Trino 439 release]({{site.url}}/docs/current/release/release-439.html), caching
+from object storage file systems got a refresh. Catalogs using the Delta Lake,
+Hive, and Iceberg connectors now get to access performance benefits from the new
+Alluxio-powered file system caching.
+
+<!--more-->
+
+## In the past
+
+So how did we get here? A long, long time ago, Qubole open-sourced a [light
+light-weight data caching framework called
+RubiX](https://github.com/qubole/rubix). The library was integrated into the
+Trino Hive connector, and it enabled [Hive connector storage
+caching]({{site.url}}/docs/438/connector/hive-caching.html). But over time, any
+open source project without active maintenance becomes stale. And like a stale
+cache, a stale open source project can cause issues, or becomes outdated and
+unsuitable for modern use. Though RubiX had once served Trino well, it was time
+to remove the dust, RubiX had to go.
```

Comment:
Grammar is off. Last part seems to be missing "and"

Reply:
yeah .. could have been "and" .. or long dash ... opted for and